hedera-node/src/test/java/com/hedera/services/txns/contract/ContractCreateTransitionLogicTest.java

```diff
@@ -218,29 +228,44 @@ void followsHappyPathWithOverrides() {
 		TransactionRecord creation = TransactionRecord.newBuilder()
 				.setReceipt(TransactionReceipt.newBuilder()
 						.setStatus(SUCCESS)
-						.setContractID(created)
+						.setContractID(contractAccount.getId().asGrpcContract())
 						.build())
-				.setContractCreateResult(ContractFunctionResult.newBuilder().setGasUsed(555))
+				.setContractCreateResult(ContractFunctionResult.newBuilder().setGasUsed(1234L))
 				.build();
 
 		givenValidTxnCtx();
 		// and:
+		given(accountStore.loadAccount(senderAccount.getId())).willReturn(senderAccount);
+		given(worldState.newContractId(senderAccount.getId().asEvmAddress())).willReturn(contractAccount.getId());
 		given(hfs.exists(bytecodeSrc)).willReturn(true);
 		given(hfs.cat(bytecodeSrc)).willReturn(bytecode);
+		given(accessor.getTxn()).willReturn(contractCreateTxn);
+		given(txnCtx.accessor()).willReturn(accessor);
+
+		var results = TransactionProcessingResult.successful(null, Optional.empty(), 1234L, Bytes.EMPTY, contractAccount.getId().asEvmAddress());
+
+		given(evmTxProcessor.execute(
+				any(),
+				any(),
+				anyLong(),
+				anyLong(),
+				any(),
+				any()))
+				.willReturn(results);
 
 		// when:
 		subject.doStateTransition();
 
 		// then:
-		verify(txnCtx).setCreateResult(creation.getContractCreateResult());
-		verify(txnCtx).setCreated(created);
-		verify(txnCtx).setStatus(SUCCESS);
+		verify(worldState).addPropertiesFor(any(), anyString(), any(), any());
+		verify(recordServices).externaliseEvmCreateTransaction(any());
```

Comment:
You should verify as-well that:
- worldState.addPropertiesFor was called exactly with the correct arguments
- worldState.reclaim was **not** called
- worldState.persist was called 
- txnCtx was called exactly with the correct argument